pkg/utils/utils.go

```diff
@@ -1,22 +1,25 @@
 package utils
 
 import (
-	"crypto/md5"
 	"fmt"
+	"github.com/ikitiki/logical_backup/pkg/message"
 	"path"
 	"strconv"
 	"strings"
 	"time"
 
 	"github.com/ikitiki/logical_backup/pkg/dbutils"
-	"github.com/ikitiki/logical_backup/pkg/message"
 )
 
-func TableDir(tbl message.NamespacedName) string {
-	tblHash := fmt.Sprintf("%x", md5.Sum([]byte(tbl.Sanitize())))
+func TableDir(tbl message.NamespacedName, oid dbutils.Oid) string {
```

Comment:
maybe we should check if `OID` is valid before calling `TableDir` function, in this case there'll be no need to pass table name to it